src/gps/GPS.cpp

```diff
@@ -1039,12 +1039,12 @@ void GPS::down()
             softsleepSupported = true;
 
         if (softsleepSupported) {
-            // How long does gps_update_interval need to be, for GPS_HARDSLEEP to become more efficient than
+            // How long does broadcast_smart_minimum_interval_secs need to be, for GPS_HARDSLEEP to become more efficient than
```

Comment:
The gps update interval and the smart update stuff is independent and used by people differently, this is not deprecated and shouldn't be removed.

Reply:
It's easy to get confused between the broadcast interval and the update interval :)